rmd_files/RNAseq_report.Rmd

```diff
@@ -4398,7 +4400,7 @@ rm(counts_vs_transformed, data_transformation_nonfiltered, data_transformation_f
 
 Table summarising **all altered genes** listed across following report sections: `r if ( runPcgrChunk ) { c("[Mutated genes], ") }` `r if ( runFusionChunk ) { c("[Fusion genes] (supported by genomic data or reported in FusionGDB), ") }` `r if ( runSVsChunk ) { c("[Structural variants], ") }` `r if ( runPurpleChunk ) { paste0("[CN altered genes] (CN values =< ", cn_bottom, " or >= ", cn_top, " and reported as cancer genes)") }` [Immune markers] or [HRD genes]. The *Resources* column contains links to databases that may provide additional source of evidence for the altered genes' clinical significance. Genes ordered by the number of report sections they appear in (*Count* column) and then alphabetically.
 
-`r if ( is.na(sunburst_plot[[2]]) ) { c("<span style=\"color:#ff0000\">NOTE</span>, no genes are listed in more then one section.") }`
+`r if ( is.na(sunburst_plot[[2]])["x"] ) { c("<span style=\"color:#ff0000\">NOTE</span>, no genes are listed in more then one section.") }`
```

Comment:
@skanwal and here.

Reply:
updated in https://github.com/umccr/RNAsum/pull/77